parastell.py

```diff
@@ -465,13 +478,52 @@ def stellarator_torus(
     return torus, cutter
 
 
+def expand_ang(ang_list, num_ang):
+    """Expands list of angles by linearly interpolating according to specified
+    number to include in stellarator build.
+
+    Arguments:
+        ang_list (list of float): user-supplied list of toroidal or poloidal
+            angles (deg).
+        num_ang (int): number of angles to include in stellarator build.
+    
+    Returns:
+        ang_list_exp (list of float): interpolated list of angles (deg).
+    """
+    # Initialize interpolated list of angles
+    ang_list_exp = []
+
+    # Compute total angular extent of supplied list
+    ang_ext = ang_list[-1] - ang_list[0]
+
+    # Compute average distance between angles to include in stellarator build
+    ang_diff_avg = ang_ext/num_ang
+    
+    # Loop over supplied angles
+    for ang, next_ang in zip(ang_list[:-1], ang_list[1:]):
+        # Compute number of angles to interpolate
+        n_ang = int(np.ceil((next_ang - ang)/ang_diff_avg))
```

Comment:
`ceil()` should always return an integer already

Reply:
`np.ceil()` returns a float. I could use the math package instead to return an integer. I just didn't want to import a package for a single function, but I could just import `math.ceil()`.

Reply:
No, what you have is fine... odd choice for `numpy` to return a float version of the integer that this method evaluates